go.mod

```diff
@@ -3,16 +3,17 @@ module go.flow.arcalot.io/podmandeployer
 go 1.21
 
 require (
-	github.com/docker/docker v24.0.7+incompatible
-	github.com/docker/go-connections v0.4.0
+	github.com/docker/docker v25.0.2+incompatible
+	github.com/docker/go-connections v0.5.0
```

Comment:
The podman deployer uses docker types? Interesting...

Reply:
It gives new depth to the notion that Podman is plug-compatible with Docker....